docs/release_notes/feb2023.md

```diff
@@ -8,6 +8,67 @@ tags:
   - changelog
 ---
 
+## 2.8.0.0 (February 28, 2023)
+
+- Prophecy Python libs version: 1.4.2
+- Prophecy Scala libs version: 7.0.7
+
+### Updates {#UpdatesRelease280}
+
+#### No team required on sign up
+
+Previously, A user was asked to enter a unique team when signing up for Prophecy. To improve the sign-up experience, requirement of providing a Team is now removed.  
+Now the user can just sign up by giving his Name, Email and Company name. He can be added to a team by Admin post sign up.
```

Comment:
his -> their
he -> They
"Admin post sign up" -> "Admin after signing up"

Reply:
Oops, Should have kept in mind the gender-neutral pronouns. Thank you for pointing out.